src/Symfony/Bundle/FrameworkBundle/Command/CachePoolListCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\HttpKernel\CacheClearer\Psr6CacheClearer;
+
+/**
+ * List available cache pools.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+final class CachePoolListCommand extends Command
+{
+    protected static $defaultName = 'cache:pool:list';
+
+    private $poolClearer;
+
+    public function __construct(Psr6CacheClearer $poolClearer)
```

Comment:
what about injecting an array of pool names from `CachePoolClearerPass` instead?

Reply:
haha, You are correct. I was being lazy. =)